docs/example/searchbarDocs.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <doc-section id="searchbarDocs" name="Searchbar">
+    <div class="bs-example">
+      <searchbar
+        :data="searchData"
+        :on-hit="consoleCallback"
+        :template="searchTemplate"
+        template-name="search" />
+    </div>
+    <doc-code language="markup">
+      <searchbar
+        :data="searchData"
+        :template="searchTemplate"
+        template-name="search" />
+    </doc-code>
+    <doc-code language="javascript">
+      <!-- eslint-disable vue/html-indent, vue/no-parsing-error, vue/valid-v-html -->
+      new Vue {
+        components: {
+          searchbar
+        },
+        data() {
+          return {
+            searchData: [
+              {
+                'headings': {
+                  'normal-include': 'Normal include',
+                  'establishing-requirements': 'Establishing Requirements',
+                  'include-segment': 'Include segment',
+                  'dynamic-include': 'Dynamic include',
+                  'boilerplate-include': 'Boilerplate include',
+                  'nested-include': 'Nested include',
+                  'html-include': 'HTML include',
+                  'include-from-another-markbind-site': 'Include from another Markbind site',
+                  'feature-list': 'Feature list',
+                },
+                'keywords': 'k1, k2, k3',
+                'src': 'index.md',
+                'title': 'Hello World',
+
+              },
+              {
+                'headings': {
+                  'popover-initiated-by-trigger-honor-trigger-attribute':
+                  'popover initiated by trigger: honor trigger attribute',
+                  'support-multiple-inclusions-of-a-modal': 'Support multiple inclusions of a modal',
+                  'remove-extra-space-in-links': 'Remove extra space in links',
+                },
+                'keywords': 'k4, k5, k6',
+                'src': 'bugs/index.md',
+                'title': 'Open Bugs',
+              },
+              {
+                'headings': {
+                  'feature-list': 'Feature list',
+                },
+                'keywords': 'k7, k8, k9',
+                'src': 'sub_site/index.md',
+                'title': '',
+              },
+            ],
+            searchTemplate: '<span v-html="item.title | highlight value"></span>'
```

Comment:
@acjh Should we just consider using this value as the default? I think the logic in searchbar now assumes that the structure of `item` will always include `title`, `keywords` and a list of `heading`. Seems to be reasonable to give the user a default search template in that case.